enclave/rsa.c

```diff
@@ -160,17 +166,22 @@ static oe_result_t _generate_key_pair(
         OE_RAISE(OE_FAILURE);
 
```

Comment:
Can you add logging messages for lines 162 and 166

Reply:
OE_RAISE already print error message by default